### PR TITLE
Redesign

### LINKED ISSUE #61: Comment Menu UI CSS issues
image

The border radius seems a bit too big. Also it doesn't seem like its "round".

### PR DESCRIPTION
**Pull request by rohanshiva**
Sorry for always redesigning, and changing up the styles. But, I promise I won't make any major changes going forward. 

Thanks to you we made a lot of progress recently, and I thought it was a good idea to clean up some of the things and just redesign with consistent styling. 

Please let me know if you have any feedback on anything, happy to change things. 

Again, I apologize for changing it up. We are getting closer and closer to launching this, so I thought it was worth it to make it look nice. 

[Demo](https://www.dropbox.com/s/fdabr585g2jnlb4/gitgame_redesign.mp4?dl=0)

> Note: right side on home page is just a placeholder. Once the app is ready, we can put some screenshots of the app. I was even thinking of a minigame demo to put there. We can discuss about this.

### CODE REVIEW

**Comment by Ramko9999** [comment]
I wonder if we should further distinguish a selected comment. What was your reasoning for removing the box shadow we had previously? I feel like a selected comment should stick out more.
image

**Comment by Ramko9999** [comment]
I liked how "svelte" the old help comments were. This feels a bit less welcoming to the eye.
image

File: client/src/components/commentSider/help/Help.tsx [hunk 1]
```diff
@@ -9,10 +9,14 @@ interface HelpCommentProps {
...
   return (
     <div className="help-card">
-      <div>
-        <Icon.Info size={"1rem"} />
-      </div>
-      {children}
+      <span>
+        <span>
+          <b>
+            {'Note: '}
```
Comment (Ramko9999) [inline client/src/components/commentSider/help/Help.tsx]: Should we replace `Note: ` with perhaps a prop called `title`? In the "Help" filter, it seems unnecessary to have `Note:`.
image

**Comment by Ramko9999** [comment]
The landing page is too sexy! I like how the buttons "float up" when you hover over them.

**Comment by Ramko9999** [comment]
I know I am responsible for this, but was curious if you have any ideas for a better-selected comment highlight color when dark mode is toggled.
image

**Comment by rohanshiva** [comment]
> I wonder if we should further distinguish a selected comment. What was your reasoning for removing the box shadow we had previously? I feel like a selected comment should stick out more.
> image
> 


That makes sense. The reason to remove the box shadow was that I felt that it was too strong with the new styles. It was not good looking, but I can certainly look into modifying the box shadow to be less intensive but still make a difference.

**Comment by rohanshiva** [comment]
> I know I am responsible for this, but was curious if you have any ideas for a better-selected comment highlight color when dark mode is toggled. image

Done

image

**Comment by rohanshiva** [comment]
> I wonder if we should further distinguish a selected comment. What was your reasoning for removing the box shadow we had previously? I feel like a selected comment should stick out more. 

image


image

Let me know if this is better.

File: client/src/components/commentSider/help/Help.tsx (hunk 1 above)
Comment (rohanshiva) [reply client/src/components/commentSider/help/Help.tsx]: Removed `Note:`

Comment (rohanshiva) [reply client/src/components/commentSider/help/Help.tsx]: Will work on making the help widget look better in another PR.

**Comment by Ramko9999** [comment]
🚀

**Pull request merged**